test/testdata/java_shelf_name.baseline

```diff
@@ -27,12 +27,17 @@ import java.util.List;
 public class ShelfName implements ResourceName {
 
   private static final PathTemplate PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding("shelves/{shelf}");
+      PathTemplate.createWithoutUrlEncoding("projects/{project}/shelves/{shelf}");
```

Comment:
All the changes in this file result from the path "shelves/{shelf}" being changed to "projects/{project}/shelves/{shelf}"